terminal/TerminalServer.cpp

```diff
@@ -153,7 +153,13 @@ void runJumpHost(shared_ptr<ServerClientConnection> serverClientState) {
           if (!serverClientState->readMessage(&message)) {
             break;
           }
-          RawSocketUtils::writeMessage(terminalFd, message);
+          try {
+            RawSocketUtils::writeMessage(terminalFd, message);
+          } catch (const std::runtime_error &ex) {
+            LOG(INFO) << "Terminal session ended";
```

Comment:
Let's log that it specifically ended because the unix socket died and also log ex.what()